<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Booking_Window</class>
 <widget class="QDialog" name="Booking_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>816</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Booking</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>567</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FiraCode Nerd Font Mono Ret</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Booking</string>
   </property>
  </widget>
  <widget class="QComboBox" name="room_booking">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="room">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FiraCode Nerd Font Mono Ret</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Room:</string>
   </property>
  </widget>
  <widget class="QLabel" name="date">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FiraCode Nerd Font Mono Ret</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Date:</string>
   </property>
  </widget>
  <widget class="QLabel" name="time_from">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FiraCode Nerd Font Mono Ret</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>From:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="date_booking">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="displayFormat">
    <string>d/M/yyyy</string>
   </property>
  </widget>
  <widget class="QComboBox" name="time_start_booking">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>280</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="time_to">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>280</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FiraCode Nerd Font Mono Ret</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>To:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="time_end_booking">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>280</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="teacher">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FiraCode Nerd Font Mono Ret</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Teacher:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="teacher_booking">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>15</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="add_btn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FiraCode Nerd Font Mono Ret</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0, 255, 0);
border-style:outset;
border-radius:9px;
border-width:2px;
border-color:black;</string>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="update_btn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FiraCode Nerd Font Mono Ret</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 0);
border-style:outset;
border-radius:9px;
border-width:2px;
border-color:black;</string>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_btn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>80</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FiraCode Nerd Font Mono Ret</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 61, 64);
border-style:outset;
border-radius:9px;
border-width:2px;
border-color:black;</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="table_booking">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>541</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="refresh_btn">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FiraCode Nerd Font Mono Ret</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(33, 75, 57);
color:white;
border-style:outset;
border-radius:9px;
border-width:2px;
border-color:black;</string>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
